Ha-ha, now build fails because of
```
Nov 07 16:53:09 /build/base/base/safeExit.cpp:14:5: error: use of undeclared identifier 'abort'
Nov 07 16:53:09     UNREACHABLE();
Nov 07 16:53:09     ^
Nov 07 16:53:09 /build/base/base/defines.h:131:31: note: expanded from macro 'UNREACHABLE'
Nov 07 16:53:09         #define UNREACHABLE() abort()
Nov 07 16:53:09                               ^
Nov 07 16:53:09 1 error generated.
```

--- a/base/base/safeExit.cpp
+++ b/base/base/safeExit.cpp
@@ -1,6 +1,7 @@
 #if defined(OS_LINUX)
 #    include <sys/syscall.h>
 #endif
+#include <cstdlib>
 #include <unistd.h>
 #include <base/safeExit.h>
 #include <base/defines.h> /// for THREAD_SANITIZER

